docker/reqmgr2ms/run.sh

```diff
@@ -94,6 +148,12 @@ MATCH_RUCIO_AUTH_URL=`cat ${SERVICE_CONFIG} | egrep '^RUCIO_AUTH_URL =' | awk -F
 sed -i -e "s,rucio_host.*,rucio_host =${MATCH_RUCIO_URL},g" ${RUCIO_CONFIG}
 sed -i -e "s,auth_host.*,auth_host =${MATCH_RUCIO_AUTH_URL},g" ${RUCIO_CONFIG}
 
+# Edit the MSUnmerged configuration file (msConfig for the service) at runtime
+# based on the set of arguments passed to the run.sh script by the service
+# statrtup command defined in the .yaml file of the currently starting service
+[[ -n $rseExpr ]] && sed -e "s/^[[:blank:]]*RSEEXPR.*/RSEEXPR = \"${rseExpr}\"/g" ${cdir}/config-unmerged.py
```

Comment:
Aren't you missing this `RSEEXPR` variable in the configuration files?

Reply:
It is there in my personal machine ... unfortunately I cannot push my changes neither to github nor to gitlab `service_config` because none the previous PR related to the changes in `config-unmerged.py` file are yet merged.

Reply:
Unless I'm missing your point, why can't you edit your PR/MR and add the RSEEXPR to the current changes? It looks much cleaner than making yet another 3 PRs to provide one more service configuration parameter.

Reply:
Let me try fitting it in those commits ... may need to tell @muhammadimranfarooqi  to wait for a minute or two before merging though.

Reply:
done